third_party/airflow/armada/operators/jobservice.py

```diff
@@ -18,6 +20,11 @@ class JobServiceClient:
     def __init__(self, channel):
         self.job_stub = jobservice_pb2_grpc.JobServiceStub(channel)
 
+    @tenacity.retry(
```

Comment:
I'd love some kind of test that can verify that this library works with GRPC.  

Can you look into adding a unit test?  You could use our mock python grpc class for jobservice.

Otherwise this is looking really good!

Reply:
The test should check whether tenacity makes retries or not , right?

Reply:
Yep.